auction-app-backend/src/main/java/com/example/auctionapp/security/JwtAuthenticationFilter.java

```diff
@@ -24,44 +24,36 @@ public class JwtAuthenticationFilter extends OncePerRequestFilter {
     private JwtService jwtService;
 
     @Autowired
-    UserDetailsServiceImpl userDetailsServiceImpl;
+    private UserDetailsServiceImpl userDetailsServiceImpl;
```

Comment:
Why are we using here Autowired and not creating constructor and setting it there, because we do that in the rest of the app